src/Models/AdminModel.php

```diff
@@ -55,31 +72,60 @@ public function setUserToAuth($user_id){
         $ans = $query->get_result();
         return $ans;
     }
-    public function deleteUser($id){
-        $query = $this->conn->prepare("DELETE FROM userauth WHERE id = ?");
-        $query->bind_param("i", $id);
+    public function deleteUser($id)
+    {
+        $query = $this->conn->prepare('DELETE FROM userauth WHERE id = ?');
+        $query->bind_param('i', $id);
         $query->execute();
         return;
     }
 
-    public function insertLogo() {
+    // funtion insert logo
+    public function updateLogo($site_name)
+    {
         $logo = $_FILES['logo_upload'];
         $file = $logo['name'];
         $file_tmp = $logo['tmp_name'];
 
-        $profile_ext = explode('.',  $file);
+        $profile_ext = explode('.', $file);
         $filecheck = strtolower(end($profile_ext));
 
-        $file_ext_stored = array('jpeg', 'png', 'jpg');
+        $file_ext_stored = ['jpeg', 'png', 'jpg'];
+
+        // location
+        $location = 'assets/images/logo/' . $file;
+        $resized_loc = 'assets/images/logo/resized_' . $file;
+
+        // file extension
+        $file_extension = pathinfo($location, PATHINFO_EXTENSION);
+        $file_extension = strtolower($file_extension);
 
         if (in_array($filecheck, $file_ext_stored)) {
-            $destinationFile = 'assets/images/logo/' . $file;
-            move_uploaded_file($file_tmp, $destinationFile);
+            $manager = new ImageManager(['driver' => 'gd']);
+
+            $img = Image::make($file_tmp)
+                ->resize(50, 50)
+                ->save($resized_loc);
+
+            move_uploaded_file($img, $resized_loc);
         }
-        $query = $this->conn->prepare("INSERT INTO config(logo) VALUES(?)");
-        $query->bind_param("s", $destinationFile);
+        $query = $this->conn->prepare(
+            'UPDATE config SET logo = ?, siteName = ? WHERE id = 22'
```

Comment:
I don't think that assigning a static value (22 to the id) to the query will be a correct solution. Because the config table on other users local machine might have some other id and not 22. So the update query might not work as there is not data having id 22.

Reply:
Yes. I will update this soon.

Reply:
@RChaubey16  updated the changes. Can you please check and let me know if it is working. Thanks.

Reply:
@Libbna Yes now it seems correct. Approving changes.